packages/airnode-deployer/src/infrastructure/index.ts

```diff
@@ -207,7 +209,7 @@ async function terraformAirnodeManage(
 }
 
 // `maxConcurrency` field is required for deployment but missing for removal. This is the easiest way to type it.
-type CloudProviderExtended = CloudProvider & {
+export type CloudProviderExtended = CloudProvider & {
```

Comment:
Do we need to export this?

Reply:
I was getting a type error without including it [here](https://github.com/api3dao/airnode/pull/1276/files#diff-13b7c9bcdda766f79084f5c09740a68e14cda9410e225e5d9bc44d643f332562R80). It's due to the nodeSettings.cloudProvider having the option of 'local', but that is not allowed once deploying.